project/Dependencies.scala

```diff
@@ -71,6 +74,24 @@ object Dependencies {
     }
   }
 
+  def json4sVersionForScala(scalaVersion: String): String = {
+    scalaVersion match {
+      case _ if scalaVersion.startsWith("2.11") => Versions.json4s_spark2
+      case _ if scalaVersion.startsWith("2.12") => Versions.json4s_spark3
```

Comment:
Please keep in Sync with Laco's Scala 2.13 support in #78

Reply:
ok, added case for scala 2.13